pkg/squid/src/handlers/control/events/memberAddedHandler.ts

```diff
@@ -8,12 +8,12 @@ import { EventHandlerContext } from '@subsquid/substrate-processor';
 async function handleMemberAddedEvent(context: EventHandlerContext) {
 	let eventName = 'Control.MemberAdded';
 	let raw_event = new ControlMemberAddedEvent(context);
-	if (!raw_event.isV61) {
+	if (!raw_event.isV62) {
```

Comment:
what is the approach for this? usually you would use version ranges like in the previous type definitions on polkadotjs and in the old repo. if it is only latest and everything else breaks...